service.py

```diff
@@ -37,9 +38,69 @@ def autostart():
         else:
             cleaner.monitor.waitForAbort(service_sleep)
 
-    log("Abort requested. Terminating.")
+    debug("Abort requested. Terminating.")
     return
 
 
+    
```

Comment:
I'm gonna write a comment here about the entire ``service.py`` file, so it's easier to read. I like the use of a ``Monitor`` to listen for notifications from the player. I have added some general comments regarding improvement below.

- I would like to ask that you use double quotes instead of single quotes, to make the code more consistent. 
- The same I would ask about using format strings: please use ``"This kind of {0} notation".format("string formatting")`` instead of ``"this %s of formatting" % "type"``. 
- Finally I would like to ask that you write detailed and specific debugging messages (in English). This helps me immensely in case a future update breaks something in your code and you are not around to help.